### ISSUE TITLE
Unable to use =RO, =RW, =NC for Branches

### ISSUE THREAD

**Issue by Artiume** [trapexit/mergerfs#744]
General description
===================
Attempting to connect a new branch using the =NC tag, but I can't seem to get it to connect in fstab. So I tried to use xattr and it still doesn't respond. I can remove the tag and the branch will connect properly. 

Actual behavior
===============
```
artiume@noder:/data/unionfs$ xattr -w user.mergerfs.srcmounts '+>/home/artiume/test=RO' .mergerfs
artiume@noder:/data/unionfs$ xattr -l .mergerfs
user.mergerfs.srcmounts: /data/bricks/node2a:/mnt/external:/data/bricks/heihei:/data/movie1:/data/movie2:/data/tv1:/data/tv2
```
The new mount doesn't appear anytime I use any of the modifier tags on new branches. 

System information
==================
Please provide as much of the following information as possible:

* [ ] mergerfs version: `mergerfs -V`
```
artiume@noder:/data/unionfs$ mergerfs -v
mergerfs version: 2.21.0
FUSE library version: 2.9.7
fusermount version: 2.9.7
using FUSE kernel interface version 7.19
```
* [ ] mergerfs settings: `cat /etc/fstab` or the command line arguments
* [ ] Linux version: `uname -a`
```
artiume@noder:/data/unionfs$ uname -a
Linux noder 4.15.0-88-generic #88-Ubuntu SMP Tue Feb 11 20:11:34 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```
* [ ] Versions of any additional software being used
* [ ] List of drives, filesystems, & sizes: `df -h`
* [ ] strace of application having problem: `strace -f -o /tmp/app.strace.txt <cmd>` or `strace -f -p <appPID> -o /tmp/app.strace.txt`
* [ ] strace of mergerfs while app tried to do it's thing: `strace -f -p <mergerfsPID> -o /tmp/mergerfs.strace.txt`

**Comment by trapexit** [comment]
**mergerfs version: 2.21.0**

You're reading new docs for old versions.

2.21.0 is a few years old. It doesn't support that feature. Upgrade to the latest as described in the docs and it'll work fine.

**Comment by Artiume** [comment]
Yup, that fixed it. Thanks :) .

I tried building from master, and I had success on my armhf devices, but my amd host ended having issues to where I had to pull the packaged deb to make it work. Here's the snippet. It might've been my sources being messed up, just wanted to point it out in case it is a real bug.

```
Reading package lists...
Building dependency tree...
Reading state information...
Package debhelper is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

Package automake is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  automake1.11

Package libtool is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

W: --force-yes is deprecated, use one of the options starting with --allow instead.
E: Package 'debhelper' has no installation candidate
E: Package 'automake' has no installation candidate
E: Package 'libtool' has no installation candidate
``

**Comment by trapexit** [comment]
What OS?

You don't have to build packages for most Debian based systems: https://github.com/trapexit/mergerfs/releases

**Comment by Artiume** [comment]
Ubuntu 18.06, it's the pc from the initial post. 

Yeah, I just tried building because I'm a bit of a masochist lol.

**Comment by trapexit** [comment]
18.04 you mean?

Do you have some extra stuff in your deb sources? I build for 18.04 all the time and certainly don't have issues installing the deb building dependencies.

**Comment by Artiume** [comment]
Yeah, 18.04 lol. 

Looks like I have some Debian sources in there. I'm not that proficient with packaging so I probably did something at some point. 

```
artiume@noder:~/mergerfs$ sudo tools/install-build-pkgs
Hit:1 http://as-repository.openvpn.net/as/debian bionic InRelease
Hit:2 https://repo.jellyfin.org/ubuntu bionic InRelease
Get:3 http://http.us.debian.org/debian stable InRelease [122 kB]
Hit:4 http://ppa.launchpad.net/projectatomic/ppa/ubuntu bionic InRelease
Hit:5 http://archive.ubuntu.com/ubuntu bionic InRelease
Err:3 http://http.us.debian.org/debian stable InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY 648ACFD622F3D138 NO_PUBKEY DCC9EFBF77E11517
Reading package lists...
W: GPG error: http://http.us.debian.org/debian stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY 648ACFD622F3D138 NO_PUBKEY DCC9EFBF77E11517
E: The repository 'http://http.us.debian.org/debian stable InRelease' is not signed.
Reading package lists...
Building dependency tree...
Reading state information...
Package debhelper is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

Package automake is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  automake1.11

Package libtool is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

W: --force-yes is deprecated, use one of the options starting with --allow instead.
E: Package 'debhelper' has no installation candidate
E: Package 'automake' has no installation candidate
E: Package 'libtool' has no installation candidate
```
